sections/kubecontext.zsh

```diff
@@ -31,6 +31,18 @@ spaceship_kubecontext() {
   local kube_context=$(kubectl config current-context 2>/dev/null)
   [[ -z $kube_context ]] && return
 
+  # Apply colors to context names that are set in keys of SPACESHIP_KUBECONTEXT_COLORGROUPS KV map.
+  # See Options.md for usage example.
+  if [[ ${#SPACESHIP_KUBECONTEXT_COLORGROUPS[@]} -gt 0 ]]; then
+    local pattern color
+    for pattern color in ${(kv)SPACESHIP_KUBECONTEXT_COLORGROUPS}; do
+      if [[ "$kube_context" == "$pattern" || "$kube_context" =~ "$pattern" ]]; then
+        kube_context="%F{$color}$kube_context%F{$color}"
```

Comment:
Yeah, that 2nd color is a mistake, it was supposed to be the section color variable there.

Maybe it would make sense to set the entire section color per group then, instead of only the contents?

Reply:
Essentially you are right, yes, we should set the color of the whole section.

Reply:
Done